modules/scheduler/gke-cluster/variables.tf

```diff
@@ -334,16 +334,20 @@ variable "rdma_subnetwork_name_prefix" {
   type        = string
 }
 
-# REMOVE_ME: It's a temporary variable used in internal testing
-variable "is_gke_sandbox" {
-  description = "Temporary variable to identify the GKE sandbox environment"
-  default     = false
-  type        = bool
+variable "cluster_reference_type" {
+  description = "How the google_container_node_pool.system_node_pools refers to the cluster. Possible values are: {SELF_LINK, NAME}"
+  default     = "SELF_LINK"
+  type        = string
+}
+
+variable "cluster_availability_type" {
```

Comment:
nit: I would just use the boolean `is_multi_zonal` as I do not think this will change that often.
but this can be done later in later PR, if you agree

Reply:
Resolving this to merge.